libs/wingsdk/src/target-tf-aws/service.ts

```diff
@@ -0,0 +1,338 @@
+            Principal: {
+              Service: "ecs-tasks.amazonaws.com",
+            },
+            Action: "sts:AssumeRole",
+          },
+        ],
+      }),
+      inlinePolicy: [{
+        name: `${this.assetName}-inline-policy`,
+        policy: JSON.stringify({
+          Version: "2012-10-17",
+          Statement: [
+            {
+              "Effect": "Allow",
+              "Action": [
+                "logs:CreateLogStream",
+                "logs:PutLogEvents",
+                "logs:CreateLogGroup"
+              ],
+              "Resource": "*"
+            },
+            {
+              "Effect": "Allow",
+              "Action": [
+                "ecr:BatchGetImage",
+                "ecr:GetDownloadUrlForLayer",
+                "ecr:GetAuthorizationToken"
+              ],
+              "Resource": "*"
+            }
+          ],
+        }),
+      }]
+    });
+
+    let taskRole = new IamRole(this, "TaskRole", {
+      assumeRolePolicy: JSON.stringify({
+        Version: "2012-10-17",
+        Statement: [
+          {
+            Effect: "Allow",
+            Principal: {
+              Service: "ecs-tasks.amazonaws.com",
+            },
+            Action: "sts:AssumeRole",
+          },
+        ],
+      }),
+      inlinePolicy: [{
+        name: `${this.assetName}-inline-policy`,
+        policy: Lazy.stringValue({
+          produce: () => {
+            this.policyStatments = this.policyStatments ?? [];
+            
+            if (this.policyStatments.length !== 0) {
+              return JSON.stringify({
+                Version: "2012-10-17",
+                Statement: this.policyStatments
+              });
+            }
+
+            return JSON.stringify({
+              Version: "2012-10-17",
+              Statement: [
+                {
+                  Effect: "Allow",
+                  Action: "none:null",
+                  Resource: "*"
+                }
+              ]
+            })
+          }
+        }),
+      }]
+    });
+
+    let logGroup = new CloudwatchLogGroup(this, "LogGroup", {
+      name: `/ecs/${this.assetName}`,
+      retentionInDays: 7,
+    });
+
+    let taskDef = new EcsTaskDefinition(this, "Task", {
+      family: this.assetName,
+      executionRoleArn: executionRole.arn,
+      taskRoleArn: taskRole.arn,
+      containerDefinitions: Lazy.anyValue({
+        produce: () => JSON.stringify([
+            {
+              name: this.assetName,
+              image: image.name,
+              environment: Object.entries(this._env).map(([key, value]) => {
+                return {
+                  name: key,
+                  value: value,
+                };
+              }
+              ),
+              logConfiguration: {
+                logDriver: "awslogs",
+                options: {
+                  "awslogs-group": `/ecs/${this.assetName}`,
+                  "awslogs-region": "us-east-1", // TODO: Can I ignore this?
```

Comment:
Alternatively you should be able to use `app.region` (if we don't have a property like this we should definitely create one and use a TF data resource to obtain the current region).

Here's how you can do this:

https://github.com/winglang/winglibs/blob/main/containers/aws.w